### DISCUSSION TITLE
Add more column sorting options and a compact list view

### DISCUSSION THREAD

**Discussion by ghost** [ciderapp/Cider#1650] [Feature Request]
I love cider so far, but one feature that's lacking is the ability to change the sort options and which columns are shown, like in iTunes. I use this extensively and, also would be nice if you could have a compact list view.
![image](https://user-images.githubusercontent.com/13620298/215829187-1e0bb279-9e13-4ad3-b9cc-56f1154d25c8.png)

**Comment by CordeliaGS** [comment]
seconded- came here to see if there was already an FR for this. Sorting the library & playlists at _least_ by artist or album is vital for any music player, and itunes' extensive sorting options are its best feature. including that would be huge. currently it looks like cider can't even sort a playlist at all?

**Comment by lgander** [comment]
+1 to this. Not only for sorting purposes, but I also want to list the year or the genre of each song.
